prebid-mobile/code-integration-android.md

```diff
@@ -36,30 +36,61 @@ to output the PrebidMobile framework for Android.
 ## Ad Unit Setup for Android
 {:.no_toc}
 
-Register Prebid Mobile ad units as early as possible in the application's lifecycle. Each ad unit has an `adUnitId` which is an arbitrary unique identifier of the developer's choice.
+Register Prebid Mobile ad units as early as possible in the application's lifecycle.  
+
+When registering a Prebid Mobile ad unit, you must replace `"PREBID-MOBILE-SLOT-ID"` with a unique user-defined identifier.  Prebid Mobile will use this identifier to determine the unique ad unit to which bid key-values will be associated.  This identifier must be unique across all registered Prebid Mobile ad units, and does not need to map to any ad unit ID defined in your ad server.
 
 The steps for using Prebid Mobile are as follows:
 
-1. Create the ad units with ad unit ids and add sizes for banner ad units.
+1. Create the ad units and add sizes for banner ad units.  Be sure to replace `"PREBID-MOBILE-SLOT-ID"` with a unique user-defined identifier.
 2. Add a server-side configuration for each ad unit to Prebid Server Adapter.
 3. Set targeting parameters for the ad units. (Optional)
 4. Set the primary adserver for the bid to either DFP or MoPub. (Primary ad server is necessary to determine the caching mechanism.)
 5. Set the Host for the bid to AppNexus or Rubicon.
 6. Register the ad units with the adapter to start the bid fetching process.
 
-### How to Create Ad Units
+### User-Replaced Macros
+
+In the code snippets below, any values that must be substituted by your developers are identified in capital letters.  These values are defined below:
+
+{: .table .table-bordered .table-striped }
+| Value | Description |
+|--------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `PREBID-SERVER-ACCOUNT-ID` | Your unique account ID with your Prebid Server vendor.  |
+| `PREBID-MOBILE-SLOT-ID` | User-defined identifier that must be unique across all registered Prebid Mobile ad units.  Note: this value does not need to map to any ad unit ID defined in your ad server. |
```

Comment:
"does not need to" or just "does not"? Making sure we're clear that this has nothing to do with the ad server ID.